scripts/flattener.py

```diff
@@ -144,7 +145,7 @@
 	'donor_sex': 'sex',
```

Comment:
@Jchaffer787 There needs to be an entry added to property map. Right now the field being outputted is called `sample_@type`. So, you'll need to add `'sample_@type': 'tissue_type',`